### PR TITLE
Add meeting artifacts 2024-10-03

### PR DESCRIPTION
**Pull request by riaankleinhans**
@renuchauhan221 

Still need one set of slides. Will add it in a another commit.

### CODE REVIEW

File: TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+* Microsoft - Alec Fernandez +
+* Nvidia - Fritz Alder
+* Red Hat - Yash Mankad   / Ram Pai +
+* TikTok - Mingshen Sun   / Yao Zhang
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Alice Lawernce (Setit Solutions)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md:51]:
```suggestion
* Alice Lawrence (Setit Solutions)
```

File: TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+* Caroline Perez-Vergas (Microsoft)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md:52]:
```suggestion
* Caroline Perez-Vargas (Microsoft)
```

File: TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md (hunk 1 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md]: Updated

File: TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md (hunk 2 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-10-03/CCC-TAC-Minutes-2024-10-03.md]: Updated

**Pull request merged**